initramfs/init.sh

```diff
@@ -28,12 +28,8 @@ UPDATE_DEVICES="/dev/mmcblk[0-9]p[0-9]"
 #uc3 : UltiController 3 (S5,S5r2,S3)
 DISPLAY_TYPE="uc3"
 UM_SPLASH="/SplashUM.fb"
-S3_SPLASH="/SplashS3.fb"
-S5_SPLASH="/SplashS5.fb"
 FB_DEVICE="/dev/fb0"
-S3_ARTNUM="0x00 0x03 0x41 0xea"
-S5_ARTNUM="0x00 0x03 0x45 0xcb"
-
+COLORADO_ARTNUM="0x00 0x03 0x78 0x34"
```

Comment:
Sweet, is this already the BoM number that Colorado will have?

Reply:
yes